build.gradle

```diff
@@ -187,15 +187,34 @@ allprojects {
     maven {
       url  "https://jitpack.io"
     }
+    flatDir {
+      dirs "$rootProject.projectDir/libs"
+    }
   }
 
   apply plugin: 'java'
 
   dependencies {
+    implementation files(
+      'consumers/build/libs/civil-sdt-consumers-0.0.1-SNAPSHOT.jar',
+      'dao/build/libs/civil-sdt-dao-0.0.1-SNAPSHOT.jar',
+      'dao-api/build/libs/civil-sdt-dao-api-0.0.1-SNAPSHOT.jar',
+      'domain/build/libs/civil-sdt-domain-0.0.1-SNAPSHOT.jar',
+      'handlers/build/libs/civil-sdt-handlers-0.0.1-SNAPSHOT.jar',
+      'handlers-api/build/libs/civil-sdt-handlers-api-0.0.1-SNAPSHOT.jar',
+      'interceptors/build/libs/civil-sdt-interceptors-0.0.1-SNAPSHOT.jar',
+      'producers/build/libs/civil-sdt-producers-0.0.1-SNAPSHOT.jar',
+      'producers-api/build/libs/civil-sdt-producers-api-0.0.1-SNAPSHOT.jar',
+      'services/build/libs/civil-sdt-services-0.0.1-SNAPSHOT.jar',
```

Comment:
bootJar {
  manifest {
    attributes 'Implementation-Title': project.name,
            'Implementation-Version': project.version
  }
  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
  archiveName = 'civil-sdt.jar'
  from {
    subprojects.collect { it.sourceSets.main.output }
  }
}

Reply:
The producers-commissioning and sdt-consumer-sample projects will need to be excluded from this as they aren't part of SDT itself.